basics/collections.md

```diff
@@ -1,4 +1,71 @@
 # Collections
 
+In this section is described the **API forwarding** feature of Mimic and how you can easily mock a real endpoint of your API service.
 
+To use this feature you must set the information about your real API service in the project details.
+You can set them during project set up or, if the the project already exists, opening the project details clicking on the menu icon on the top of the window and selecting _project settings_
 
+![Project Setting](../Images/project_settings.png 'Project Setting')
+
+The project settings popup allows you to set the information about your real API service:
+
+![Project Details](../Images/edit_project.png 'Project Details')
+
+We can use the following API service as example to explain how the API forwarding feature of Mimic works:
+
+[restcountries.eu](https://restcountries.eu/)
+
+It is a free and open source REST service to get informations about countries of the World.
+
+Let's create our project to mock that service API. Create a new project called **countries**:
+
+![API Forwarding Exampre Project](../Images/api_forwarding_example_project.png 'API Forwarding Exampre Project')
+
+During the set up process you must add the information about the real API service, in this case you can set:
+
+- _Domain_ : restcountries.eu
+- _Path_ : /rest/v2/
+
+You can leave empty the **port** field becasue there is not any information about it. The **path** is set to `/rest/v2/` because all the endpoints have this path in their URLs.
+
+Clicking on **Add Project** your _countries_ project is ready and it is running.
+
+## API Forwarding
+
+Your project hasn't any endpoints but it is connected to a real API service. As you know the url of your mock server is:
+
+`http://localhost:3000/countries`
+
+and the real API service has the `all` endpoint to get all data of all countries:
+
+`GET https://restcountries.eu/rest/v2/all`
+
+Let's try to do a request to the mock server for this endpoint using the following command on your Terminal:
+
+`curl http://localhost:3000/countries/all`
+
+Because the `coutries` project hasn't the `all` endpoint, this request is forwarded to the real API service using the data you set in the project details. In that way you receive the response from the real API service through Mimic.
```

Comment:
[typo] 

```suggestion
Because the `countries` project doesn't have the `all` endpoint, this request is forwarded to the real API service using the data you set in the project details. You will receive the response from the real API service proxied through Mimic.
```